README.md

```diff
@@ -1,18 +1,29 @@
 # .NET Core Samples
 
 This folder contains all the sample code that is part of any topic under
-the .NET Core documentation. There are several different projects that
+the .NET documentation. There are several different projects that
 are organized in sub-folders. These sub-folders are organized similar
-to the organization of the docs for .NET Core.
+to the organization of the docs for .NET.
 
 Some of the articles will have more than one sample associated with them. 
 
 The readme.md file for each sample will refer to the article so that
 you can read more about the concepts covered in each sample.
 
+There are two classes of samples in this repository:
+
+- **Samples** represent programs that demonstrate apoplication or library scenarios. These samples are typically larger, and often use more than one technology, feature, or toolkit.
+- **Snippets** represent small focused examples that demonstrate one feature or syntax. These should be no more than a single screen of code.
+
+In both cases, we verify the samples in our CI system.
+
+## Snippets
+
+Snippets are extracted from small programs that include the snippet. Snippets are all located in the top level **/snippets** folder. Snippets should follow all other sample guidance.
```

Comment:
The only confusing part about this is that we have snippets that do require VS like WPF snippets. So if they follow the sample guidance where we only mention .NET Core projects. they might get confused.

Reply:
I tried to rework this section. It's probably worth another review.